spec/system/planning_applications/summary_of_works_spec.rb

```diff
@@ -84,9 +85,14 @@
       within("#assessment-information-tasks") do
         expect(page).to have_content("In progress")
       end
```

Comment:
I guess we may not need this specs if you are doing `expect(check_and_assess_list_item).to have_content("In progress")` anyway?

Reply:
Surely you can't check too many tags 🤣

Reply:
There's a few examples of these scattered around. I guess this one is checking extra that it's within the id`"#assessment-information-tasks"`

Reply:
Ahh my bad ok lol